docs/guide/troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting
+
+See [rollup's troubleshooting](https://rollupjs.org/guide/en/#troubleshooting) too.
+
+If you didn't make it work, please try posting questions on [GitHub Discussions](https://github.com/vitejs/vite/discussions) or discussioning the issue in #help channel on [Vite Land Discord](https://chat.vitejs.dev).
+
+## CLI
+
+### `Error: Cannot find module 'C:\foo\bar&baz\vite\bin\vite.js'`
+
+Probably the path to your project folder includes `?`.
+There is a bug in `npm` when using Windows and the path including `?` ([npm/cmd-shim#45](https://github.com/npm/cmd-shim/issues/45)).
```

Comment:
```suggestion
The path to your project folder may include `?`, which doesn't work with `npm` on Windows ([npm/cmd-shim#45](https://github.com/npm/cmd-shim/issues/45)).
```

Simplified this part a bit.

I think we should also move single paragraphs all in the same line as it's common for markdown files to line wrap in the IDE.